bigtable/docs/snippets_table.py

```diff
@@ -403,5 +403,178 @@ def test_bigtable_table_row():
 
 
-if __name__ == "__main__":
+def test_bigtable_column_family_name():
+    # [START bigtable_column_family_name]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    column_families = table.list_column_families()
+    column_family_obj = column_families[COLUMN_FAMILY_ID]
+    column_family_name = column_family_obj.name
+    # [END bigtable_column_family_name]
+    import re
+    _cf_name_re = re.compile(r'^projects/(?P<project>[^/]+)/'
+                             r'instances/(?P<instance>[^/]+)/tables/'
+                             r'(?P<table>[^/]+)/columnFamilies/'
+                             r'(?P<cf_id>[_a-zA-Z0-9][-_.a-zA-Z0-9]*)$')
+    assert _cf_name_re.match(column_family_name)
+
+
+def test_bigtable_create_update_delete_column_family():
+    # [START bigtable_create_column_family]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    column_family_id = "column_family_id1"
+    gc_rule = column_family.MaxVersionsGCRule(2)
+    column_family_obj = table.column_family(column_family_id,
+                                            gc_rule=gc_rule)
+    column_family_obj.create()
+
+    # [END bigtable_create_column_family]
+    column_families = table.list_column_families()
+    assert column_families[column_family_id].gc_rule == gc_rule
+
+    # [START bigtable_update_column_family]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    # Already existing column family id
+    column_family_id = "column_family_id1"
+    # Define the GC rule to retain data with max age of 5 days
+    max_age_rule = column_family.MaxAgeGCRule(datetime.timedelta(days=5))
+    column_family_obj = table.column_family(column_family_id,
+                                            gc_rule=max_age_rule)
+    column_family_obj.update()
+    # [END bigtable_update_column_family]
+
+    updated_families = table.list_column_families()
+    assert updated_families[column_family_id].gc_rule == max_age_rule
+
+    # [START bigtable_delete_column_family]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    column_family_id = "column_family_id1"
+    column_family_obj = table.column_family(column_family_id)
+    column_family_obj.delete()
+    # [END bigtable_delete_column_family]
+    column_families = table.list_column_families()
+    assert column_family_id not in column_families
+
+
+def test_bigtable_create_MaxVersionsGCRule():
```

Comment:
For consistency with other samples, can you put these in the following order and use these column family names:
cf1 with MaxAge GC Rule
cf2 with Max Versions GC Rule
cf3 with union GC rule
cf4 with Intersection GC rule
cf5 with a Nested GC rule (could you add this one similar to this sample https://github.com/GoogleCloudPlatform/python-docs-samples/blob/18f766ab2e84ea106c2fbedec2382f2061204934/bigtable/tableadmin/tableadmin.py#L151)